packages/base-types/src/request/index.ts

```diff
@@ -114,6 +125,20 @@ export interface NodeButton {
   buttons?: AnyRequestButton[];
 }
 
+export const hasActionPayload = (request: BaseRequest): request is { type: string; payload: ActionPayload } =>
+  'payload' in request &&
+  typeof request.payload === 'object' &&
+  request.payload !== null &&
+  'actions' in request.payload &&
+  Array.isArray(request.payload.actions);
+
+export const hasLabelPayload = (request: BaseRequest): request is { type: string; payload: LabelRequestPayload } =>
+  'payload' in request &&
+  typeof request.payload === 'object' &&
+  request.payload !== null &&
+  'label' in request.payload &&
+  typeof request.payload.label === 'string';
```

Comment:
These type-guards are useful to resolve type issues in `general-runtime`. 

Some code needs to access `request.payload.actions` or `request.payload.label` but we don't ultimately care what type of request it is, so long as `.actions` and `.label` are defined properly. 

Therefore, these type-guards are useful to detecting partial request types.